src/styles/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  flex-wrap: wrap;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: var(--border-radius);
+}
+
+.card__product-photo {
+  margin: 0 auto 40px;
```

Comment:
You can add align-items: center rule to your card to avoid adding margin: 0 auto to each card element